docs/en_us/platform_api/source/course_catalog/catalog.rst

```diff
@@ -0,0 +1,228 @@
+########################################
+Course Catalog API Catalog Resource
+########################################
+
+With the Course Catalog API **Catalog** resource, you can complete the
+following tasks.
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+.. _Get a List of All Catalogs:
+
+***************************
+Get a List of All Catalogs
+***************************
+
+.. class enrollment.views.EnrollmentView(**kwargs)[source] (?)
```

Comment:
What is this?  Im not as familiar with RST, but `enrollment.views` seems like it might be a copy+paste error